parsec/sequester_crypto.py

```diff
@@ -1,167 +0,0 @@
-# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPL-3.0 2016-present Scille SAS
-from __future__ import annotations
```

Comment:
should oscrypto be removed from the `pyproject.toml`'sdependencies ?

(or should it be moved into de dev dependencies because e.g. it is still used by the test ?)

Reply:
It is still used in test and there:
```
if SSL_LIBRARY_PATH and CRYPTO_LIBRARY_PATH:
    import oscrypto

    oscrypto.use_openssl(
        libcrypto_path=CRYPTO_LIBRARY_PATH,
        libssl_path=SSL_LIBRARY_PATH,
    )
``` this should be removed :thinking:

Reply:
I will remove it from `parsec.__init__.py` and move the line in `pyproject.toml` under dev dependencies.

Reply:
Fixed in [ccfed32](https://github.com/Scille/parsec-cloud/pull/3629/commits/ccfed32ea9e051b927c045382b71281b8d911452)